{"uid":"b38f6910154e878d","name":"Login into the site succesfully","fullName":"Login into the site succesfully","historyId":"Login into the site succesfully","time":{"start":1677727430519,"stop":1677727430524,"duration":5},"status":"broken","statusMessage":"An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session/868ef3810cb8c89d0e54880adc75a84e/screenshot. The exception message was: Connection refused (localhost:4444) -> Connection refused (localhost:4444) -> Connection refused","statusTrace":"OpenQA.Selenium.WebDriverException: An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session/868ef3810cb8c89d0e54880adc75a84e/screenshot. The exception message was: Connection refused (localhost:4444)\n ---> System.Net.Http.HttpRequestException: Connection refused (localhost:4444)\n ---> System.Net.Sockets.SocketException (111): Connection refused\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\n   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|277_0(AwaitableSocketAsyncEventArgs saea, ValueTask connectTask, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)\n   --- End of inner exception stack trace ---\n   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.AddHttp11ConnectionAsync(HttpRequestMessage request)\n   at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.GetHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)\n   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.MakeHttpRequest(HttpRequestInfo requestInfo)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   --- End of inner exception stack trace ---\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   at OpenQA.Selenium.WebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)\n   at OpenQA.Selenium.WebDriver.GetScreenshot()\n   at SeleniumTest.Tests.Hooks.Hooks.CaptureScreenshot() in /home/sch/actions-runner/_work/GUI-Framework/GUI-Framework/3.Tests/UI/Hooks/Hooks.cs:line 176\n   at TechTalk.SpecFlow.Bindings.BindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)\n   at Allure.SpecFlowPlugin.AllureBindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"BeforeFeature [10000]","time":{"start":1677727430508,"stop":1677727430518,"duration":10},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"CreateProject [1]","time":{"start":1677727430518,"stop":1677727430518,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"CreateItems [2]","time":{"start":1677727430519,"stop":1677727430519,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"BeforeScenario [10000]","time":{"start":1677727430519,"stop":1677727430519,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"broken","statusMessage":"An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session/868ef3810cb8c89d0e54880adc75a84e/screenshot. The exception message was: Connection refused (localhost:4444) -> Connection refused (localhost:4444) -> Connection refused","statusTrace":"OpenQA.Selenium.WebDriverException: An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session/868ef3810cb8c89d0e54880adc75a84e/screenshot. The exception message was: Connection refused (localhost:4444)\n ---> System.Net.Http.HttpRequestException: Connection refused (localhost:4444)\n ---> System.Net.Sockets.SocketException (111): Connection refused\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\n   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|277_0(AwaitableSocketAsyncEventArgs saea, ValueTask connectTask, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)\n   --- End of inner exception stack trace ---\n   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.AddHttp11ConnectionAsync(HttpRequestMessage request)\n   at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.GetHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)\n   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.MakeHttpRequest(HttpRequestInfo requestInfo)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   --- End of inner exception stack trace ---\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   at OpenQA.Selenium.WebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)\n   at OpenQA.Selenium.WebDriver.GetScreenshot()\n   at SeleniumTest.Tests.Hooks.Hooks.CaptureScreenshot() in /home/sch/actions-runner/_work/GUI-Framework/GUI-Framework/3.Tests/UI/Hooks/Hooks.cs:line 176\n   at TechTalk.SpecFlow.Bindings.BindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)\n   at Allure.SpecFlowPlugin.AllureBindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)","steps":[{"name":"Given  the user navigates to the URL","time":{"start":1677727430519,"stop":1677727430520,"duration":1},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":1,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"CaptureScreenshot [10000]","time":{"start":1677727430524,"stop":1677727430525,"duration":1},"status":"broken","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"AfterFeature [10000]","time":{"start":1677727430537,"stop":1677727430537,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"thread","value":"NonParallelWorker"},{"name":"host","value":"sch"},{"name":"feature","value":"User"},{"name":"tag","value":"Smoke"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":8,"unknown":0,"total":9},"items":[{"uid":"b1c5201a9cba1ec2","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/732//#testresult/b1c5201a9cba1ec2","status":"passed","time":{"start":1677726615800,"stop":1677726621838,"duration":6038}},{"uid":"7bf9d119680a8281","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/730//#testresult/7bf9d119680a8281","status":"passed","time":{"start":1677724940416,"stop":1677724945988,"duration":5572}},{"uid":"f7e946fef2fb23c1","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/14//#testresult/f7e946fef2fb23c1","status":"passed","time":{"start":1677718799562,"stop":1677718805457,"duration":5895}},{"uid":"5f4ad0d3fc189c9e","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/729//#testresult/5f4ad0d3fc189c9e","status":"passed","time":{"start":1677714576262,"stop":1677714582028,"duration":5766}},{"uid":"174f95357b3c2739","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/728//#testresult/174f95357b3c2739","status":"passed","time":{"start":1677713205191,"stop":1677713211047,"duration":5856}},{"uid":"a9fcbec76e99a030","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/727//#testresult/a9fcbec76e99a030","status":"passed","time":{"start":1677710665610,"stop":1677710671446,"duration":5836}},{"uid":"c7feb8fe421a4a92","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/726//#testresult/c7feb8fe421a4a92","status":"passed","time":{"start":1677709896498,"stop":1677709902621,"duration":6123}},{"uid":"927014c24692de6b","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/725//#testresult/927014c24692de6b","status":"passed","time":{"start":1677709355033,"stop":1677709360964,"duration":5931}}]},"tags":["Smoke"]},"source":"b38f6910154e878d.json","parameterValues":[]}